Os/TaskCommon.cpp

```diff
@@ -1,11 +1,10 @@
 #include <Os/Task.hpp>
 #include <Fw/Types/Assert.hpp>
-#include <Fw/Logger/Logger.hpp>
-#include <cstring>
-#include <limits>
+#include <FpConfig.hpp>
+
 
 namespace Os {
-    const NATIVE_UINT_TYPE Task::TASK_DEFAULT = std::numeric_limits<NATIVE_UINT_TYPE>::max();
+    const NATIVE_UINT_TYPE Task::TASK_DEFAULT = FpLimits::PlatformUIntType_MAX;
```

Comment:
Should we change this to PlatformUIntType for clarity?

Reply:
No, as it will make refining these types properly more difficult when we get to it.